TrustModel.md

```diff
@@ -0,0 +1,426 @@
+maintaining the cluster.
+
+### Workload Provider
+This persona has control over the code, containers and pods used to provide the Application or 
+Workload.
+They define and build an application or solution to provide business value to customers and 
+clients.
+The workload provider and end user could be part of same company/organisation but 
+following the least privilege principle the application provider should not be able to view or 
+manipulate end user data without informed consent.
+- Owner of business logic containers. These may contain proprietary algorithms, models or secrets
+- Needs to prove to customer aspects of compliance
+- Defines what the solution requires in order to run and maintain compliance (resources, utility 
+  containers/services,  storage)
+- provide the protected bootimage, or designate a trusted party to provide the protected 
+  bootimage; 
+- provide the attestation verification service, or designate a trusted 
+  party  to provide the attestation verification service.
+
+Example : 3rd party software vendor
+
+### Data Owner
+Owner of data used, and manipulated by the application.
+- concerned with visibility and integrity of their data
+- concerned with compliance and protection of their data
+- uses and shares data with solutions 
+- least privilege principle means no visibility or manipulation of data is possible by 
+  Orchestration Operator or Cloud Operator personas
+
+Example : The end user/customer for the application
+
+### Persona Trust
+
+The key trust relationship here is between the Data Owner and the other personas. The Data Owner 
+trusts the code provided by the workload provider to operate across their data, however they do 
+not trust the Orchestration Operator or Cloud Operator with their data and wish to ensure 
+data confidentiality.
+
+The workload provider must not trust the Orchestration Operator or Cloud Operator  
+otherwise they risk loosing the trust of the data owner. This means the workload provider must take 
+steps to ensure the integrity of the code and that the data is appropriately protected, they may also 
+wish to maintain confidentiality of their code from other personas.
+
+
+## Lifecycle 
+The Cloud Native security Paper [^2] describes the lifecycle relating to workloads in four phases 
+- Develop
+- Distribute
+- Deploy
+- Runtime
+
+When we consider confidential containers there are lifecycle phases required before arriving at 
+the runtime environment in which the workload is truely confidential. Threats can exist during 
+these lifecycle phases and trust must be considered in the context of the lifecycle of the 
+environment supporting confidential containers.
+
+
+![Lifecycle Phases](./images/LifecyclePhases.svg)
+
+Taking the four phases above we can map this as :
+
+#### Develop
+ - Preparation and build of container images
+ - Preparation and build of the trusted boot image for the TEE. 
```

Comment:
Let's be clear that trusted boot image is only one possible implementation.

Reply:
I assume the trusted boot image is measured and can be further verified, right?

Reply:
Ah I confused trusted boot image for protected boot image. This is fine.